src/test/resources/changesets/35_delete_client_scope_mappers.yml

```diff
@@ -0,0 +1,15 @@
+id: delete-client-scope-mappers
+author: janunld
+realm: integ-test
+changes:
+  - addClientScope:
```

Comment:
migration `34_add_client_scope_mappers.yml ` is executed before. There is already a clientScope with the name `testMappers` added in the previous migration so this migration will crash. There is currently no action supporting the `deleteClientScope` usecase. Your available option are to remote the addClientScope action here or implement a deleteClientScope action and use it before this one :)